controllers/complianceeventsapi/migrations/000001_compliance_history_initial_tables.up.sql

```diff
@@ -0,0 +1,57 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS clusters(
+   id serial PRIMARY KEY,
+   name TEXT NOT NULL,
+   cluster_id TEXT UNIQUE NOT NULL,
+   UNIQUE (name, cluster_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_clusters_name ON clusters (name);
+
+CREATE TABLE IF NOT EXISTS parent_policies(
+   id serial PRIMARY KEY,
+   name TEXT NOT NULL,
+   categories TEXT [],
+   controls TEXT [],
+   standards TEXT [],
+   UNIQUE (name, categories, controls, standards)
+);
+
+CREATE TABLE IF NOT EXISTS policies(
+   id serial PRIMARY KEY,
+   kind TEXT NOT NULL,
+   api_group TEXT NOT NULL,
+   name TEXT NOT NULL,
+   parent_policy_id INT NOT NULL,
+   spec TEXT,
+   -- SHA1 hash
+   spec_hash CHAR[40],
+   CONSTRAINT fk_parent_policy_id
+      FOREIGN KEY(parent_policy_id) 
+	  REFERENCES parent_policies(id),
+   UNIQUE (kind, api_group, name, parent_policy_id, spec_hash)
+);
+
+CREATE INDEX IF NOT EXISTS idx_policies_spec_hash ON policies (spec_hash);
+
+CREATE TABLE IF NOT EXISTS compliance_events(
```

Comment:
I'm wondering about compliance operator which could have a standard, category control for the overall policy, but compliance events may be for more specific categories and controls due to the specific rule failure.  Just wondering if we may want to account for a scenario like that.